process/validatorStatisticsProcessor.go

```diff
@@ -9,8 +9,11 @@ import (
 	"github.com/multiversx/mx-chain-proxy-go/data"
 )
 
-// ValidatorStatisticsPath represents the path where an observer exposes his validator statistics data
-const ValidatorStatisticsPath = "/validator/statistics"
+const (
+	// ValidatorStatisticsPath represents the path where an observer exposes his validator statistics data
+	ValidatorStatisticsPath = "/validator/statistics"
```

Comment:
does `ValidatorStatisticsPath` need to be exported?

Reply:
No need for it to be exported. Thought that I'd leave it as it is, but I'll fix it in this PR.